src/Behat/Gherkin/Exception/CacheException.php

```diff
@@ -10,13 +10,11 @@
 
 namespace Behat\Gherkin\Exception;
 
-use RuntimeException;
```

Comment:
I guess that's the [global_namespace_import](https://cs.symfony.com/doc/rules/import/global_namespace_import.html) rule, from the Symfony ruleset.

I don't mind it either way - but be good to hear other opinions, perhaps?

Reply:
I imagine we'll eventually want Behat/Behat to be consistent with this decision too - @carlos-granados do you have a preference?

If we do decide to configure it to import everything, I guess we should just disable the rule in this PR rather than change these ones and then change them back.

Reply:
My personal preference is to import everything, it makes the code less cluttered

Reply:
@uuf6429 I make that 2 in favour of import everything, 2 fine either way. I suggest disabling the rule in this PR and reverting the changes it produced, then adding a separate PR to add it (and @ttomdewit can feed into that if he feels strongly about it)?